lib/msal-browser/FAQ.md

```diff
@@ -1,17 +1,158 @@
+1. [Where is the authority string on Azure AD Portal?](#where-is-the-authority-domain-string-on-azure-ad-portal)
+
+**[Tokens](#Tokens)**
+
+1. [How do I acquire an access token? How do I use it?](#how-do-i-acquire-an-access-token-how-do-i-use-it)
+1. [How do I renew tokens with MSAL.js?](#how-do-i-renew-tokens-with-msaljs)
+1. [How can I acquire tokens faster?](#how-can-i-acquire-tokens-faster)
+1. [I'm seeing scopes openid, profile, email, offline_access in my tokens, even though I haven't requested them. What are they?](#im-seeing-scopes-openid-profile-email-offline_access-and-userread-in-my-tokens-even-though-i-havent-requested-them-what-are-they)
+1. [How long do tokens last? How long are they valid for?](#how-long-do-tokens-last-how-long-are-they-valid-for)
+1. [What are the differences between supported audiences and account types?](#what-are-the-differences-between-supported-audiences-and-account-types)
+
+**[Scopes & Resources](Scopes-&-Resources)**
+
+1. [My application has multiple resources it needs to access to. How should I handle scopes for access tokens?](#my-application-has-multiple-resources-it-needs-to-access-to-how-should-i-handle-scopes-for-access-tokens)
+
+
+**Common Issues**
+1. [Why is MSAL throwing an error?](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-browser/docs/errors.md)
+
+***
+
+# Compatibility
+
+## What browsers are supported by MSAL.js?
+
+MSAL.js has been tested with the following browsers:
 
-# Does this library work for iframed applications?
+IE 11, Edge, Chrome, Firefox and Safari
+
+Keep these steps in mind when [using MSAL.js with IE](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/Using-msal.js-with-Internet-Explorer).
+
+There are certain known issues and mitigations documented for Safari, IE and Edge. Please check out:
+* [Known issues on IE and Edge](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/Known-issues-on-IE-and-Edge-Browser)
+* [Known issue on Safari](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/Known-issue-on-Safari)
+
+## I am moving from MSAL.js 1.x to MSAL.js to 2.x. What should I know?
+
+Please refer to our migration guide [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/v1-migration.md).
+
+## Does this library work for iframed applications?
 
 We are currently working on support for iframed applications as well as solutions for applications affected by ITP 2.x changes. You can monitor the first of those tickets [here](#1410).
 
-# Will MSAL 2.x support B2C?
+## Will MSAL 2.x support B2C?
 
 We are currently working with the B2C service team to allow for authorization code flow to work in the browser with B2C tenants. We hope to have this available shortly after release.
 
-# Common Error Handling 
+# Authentication
+
+## I don't understand the redirect flow. How does the `handleRedirectPromise` function work?
```

Comment:
nit: Do you want to add a visual representation at some point?

Reply:
Yes, it's on my list but will take it later.